Applications/AlgorithmPlaza.qmd

```diff
@@ -5,108 +5,112 @@ aliases:
 
-Once your algorithm is exposed as a service, users can 'invoke'  it with a given set of parameters. 
+Once an algorithm is exposed as a service, other users can readily access and use it for their workflows. 
+ 
 
 ## Service maturity levels
 
-To ensure quality control across the various services offered in the openEO algorithm plaza, the platform assigns a maturity level to them. This level indicates what end users can expect from the services with in terms of:
+The platform assigns maturity levels to maintain quality control across the services available in the openEO algorithm plaza. These levels indicate the expected performance and reliability of the services for end users:
 
-* Validation of the results
-* Stability
-* Scalability
-* Documentation
+* Validation of the result ensures the accuracy and reliability of the outcomes.
+* Stability shows how well the service performs under different conditions. 
+* Scalability informs users on how increasing amounts of data or users affect the algorithm's performance. 
+* Documentation provides detailed information about the service, including its purpose, inputs, outputs, and how to use it.
 
-The table below provides an overview of the different maturity levels that are applied within the openEO algorithm plaza.
+These criteria classify services into five maturity levels: Prototype, Incubating, Verified, Validated, and Operational.
 
 | Level | Description |
 | --- | --- |
-| _**Prototype**_ | Service is provided ‘as-is’, with a short description and possibly a reference to what it tries to implement. |
-| _**Incubating**_ | Quality of the service is documented with example requests (sets of parameters) and the corresponding output, as well as the resources required to generate that output. Allowing interested users to self-assess whether this service is suitable for usage. |
-| _**Verified**_ | The service is labelled verified based on its software readiness and irrelevance to the scientific validation report. |
-| _**Validated**_ | The service is validated, and validation reports are available in addition to being verified.
-| _**Operational**_ | The service has been shown to be fit for larger scale production and integration in operational systems. Rules for estimating resource usage are available, or a unit cost is established. (€ per hectare, € per request, etc.) |
+| _**Prototype**_ | Service is provided ‘as-is’, with a short description and possibly a reference to what is implemented. |
+| _**Incubating**_ | The service is documented with example requests (sets of parameters), the corresponding output, and the resources required to generate that output. |
+| _**Verified**_ | The service is labelled verified based on its software readiness and verification that validation reports are not required. |
```

Comment:
still very confusing that you intermix verified and validated.

I do not understand how a service would go from incubating to Verified. 

Please be precise.